sos/report/plugins/ceph_mds.py

```diff
@@ -75,7 +75,7 @@ def setup(self):
             "counter schema",
             "damage ls",
             "dump loads",
-            "dump tree",
+            "dump tree /",
```

Comment:
curious question, would `dump tree` work on older ceph, and now it is `dump tree /`?

Reply:
I'm not sure. I noticed the issue when reviewing a sosreport from Quincy.